src/mmw/js/src/geocode/views.js

```diff
@@ -263,6 +267,12 @@ var SearchBoxView = Marionette.LayoutView.extend({
         App.map.set('selectedGeocoderArea', null);
     },
 
+    resetIfSelectedAreaIsCleared: function(model, selectedGeocoderArea, options) {
```

Comment:
`options` is defined but never used, causing build errors.

Reply:
Thanks.